SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -115,7 +115,14 @@ void SpatialLoadBalanceEnforcer::ProcessQueuedAclAssignmentRequests()
 		if (AuthorityIntentComponent == nullptr)
 		{
 			// TODO(zoning): Not sure whether this should be possible or not. Remove if we don't see the warning again.
-			UE_LOG(LogSpatialLoadBalanceEnforcer, Warning, TEXT("(%s) Entity without AuthIntent component will not be processed. EntityId: %lld"), *WorkerId, Request.EntityId);
+			UE_LOG(LogSpatialLoadBalanceEnforcer, Warning, TEXT("Entity without AuthIntent component will not be processed. EntityId: %lld"), Request.EntityId);
+			CompletedRequests.Add(Request.EntityId);
+			return;
+		}
+
+		if (AuthorityIntentComponent->VirtualWorkerId == SpatialConstants::INVALID_VIRTUAL_WORKER_ID)
```

Comment:
We can get into a loop where an entity is queued with an invalid worker ID because the mappings weren't ready. Remove it from the list so it can be added later with a valid worker ID.

Reply:
How can this happen? afaik entities can only be spawned by the GDK once workers are ready ( -> knowing their local virtual worker ID) and should never be set to 0 after?

Reply:
I tracked this down now. It was happening because CreateEntity was being processed as part of SpatialNetDriver's tick processing, even before the SpatialNetDriver (and hence the Translator) was ready. I've added a fix there and made this Warning message a bit scarier. If we don't see it in a few weeks, I'm happy to remove the log.

Reply:
sgtm